<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    qproperty-alignment:  'AlignVCenter | AlignRight';
    border: 1px solid gray;
}

background-color: white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(215,215,215);
    border: 1px solid gray;
}
QPushButton:pressed{
    background-color: qlineargradient{x1:0,y1:0,x2:0,y2:1,
                                          stop:0 #BEBEBE, stop1: #D7D7D7};
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plusMinus">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(215,215,215);
    border: 1px solid gray;
}
QPushButton:pressed{
    background-color: qlineargradient{x1:0,y1:0,x2:0,y2:1,
                                          stop:0 #BEBEBE, stop1: #D7D7D7};
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(215,215,215);
    border: 1px solid gray;
}
QPushButton:pressed{
    background-color: qlineargradient{x1:0,y1:0,x2:0,y2:1,
                                          stop:0 #BEBEBE, stop1: #D7D7D7};
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(255,151,57);
    color: white;
    border: 1px solid gray;
}
QPushButton:pressed{
    background-color: qlineargradient{x1:0,y1:0,x2:0,y2:1,
                                          stop:0 #FF7832 stop1: #FF9739};
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 1px solid gray;
}
QPushButton:pressed{
    background-color: qlineargradient{x1:0,y1:0,x2:0,y2:1,stop:0 #dadbde, stop1: #f6f7fa};
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 1px solid gray;
}
QPushButton:pressed{
    background-color: qlineargradient{x1:0,y1:0,x2:0,y2:1,stop:0 #dadbde, stop1: #f6f7fa};
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiply">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(255,151,57);
    color: white;
    border: 1px solid gray;
}
QPushButton:pressed{
    background-color: qlineargradient{x1:0,y1:0,x2:0,y2:1,
                                          stop:0 #FF7832 stop1: #FF9739};
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 1px solid gray;
}
QPushButton:pressed{
    background-color: qlineargradient{x1:0,y1:0,x2:0,y2:1,stop:0 #dadbde, stop1: #f6f7fa};
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 1px solid gray;
}
QPushButton:pressed{
    background-color: qlineargradient{x1:0,y1:0,x2:0,y2:1,stop:0 #dadbde, stop1: #f6f7fa};
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 1px solid gray;
}
QPushButton:pressed{
    background-color: qlineargradient{x1:0,y1:0,x2:0,y2:1,stop:0 #dadbde, stop1: #f6f7fa};
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(255,151,57);
    color: white;
    border: 1px solid gray;
}
QPushButton:pressed{
    background-color: qlineargradient{x1:0,y1:0,x2:0,y2:1,
                                          stop:0 #FF7832 stop1: #FF9739};
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 1px solid gray;
}
QPushButton:pressed{
    background-color: qlineargradient{x1:0,y1:0,x2:0,y2:1,stop:0 #dadbde, stop1: #f6f7fa};
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 1px solid gray;
}
QPushButton:pressed{
    background-color: qlineargradient{x1:0,y1:0,x2:0,y2:1,stop:0 #dadbde, stop1: #f6f7fa};
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 1px solid gray;
}
QPushButton:pressed{
    background-color: qlineargradient{x1:0,y1:0,x2:0,y2:1,stop:0 #dadbde, stop1: #f6f7fa};
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(255,151,57);
    color: white;
    border: 1px solid gray;
}
QPushButton:pressed{
    background-color: qlineargradient{x1:0,y1:0,x2:0,y2:1,
                                          stop:0 #FF7832 stop1: #FF9739};
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 1px solid gray;
}
QPushButton:pressed{
    background-color: qlineargradient{x1:0,y1:0,x2:0,y2:1,stop:0 #dadbde, stop1: #f6f7fa};
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 1px solid gray;
}
QPushButton:pressed{
    background-color: qlineargradient{x1:0,y1:0,x2:0,y2:1,stop:0 #dadbde, stop1: #f6f7fa};
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(255,151,57);
    color: white;
    border: 1px solid gray;
}
QPushButton:pressed{
    background-color: qlineargradient{x1:0,y1:0,x2:0,y2:1,
                                          stop:0 #FF7832 stop1: #FF9739};
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(215,215,215);
    border: 1px solid gray;
}
QPushButton:pressed{
    background-color: qlineargradient{x1:0,y1:0,x2:0,y2:1,
                                          stop:0 #BEBEBE, stop1: #D7D7D7};
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
